examples/random_forest_example.py

```diff
@@ -0,0 +1,169 @@
+                                    },'result': True
+                                }
+                            }
+                        },'target_dimension': 'bands'
+                    }
+                },'renamelabels1': {
+                'process_id': 'rename_labels','arguments': {
+                    'data': {'from_node': 'applydimension2'},'dimension': 'bands',
+                    'target': ['B03_t0','B03_t1','B03_t2','B03_t3','B03_t4','B04_t0','B04_t1','B04_t2','B04_t3',
+                               'B04_t4']
+                    }
+                },'aggregatespatial1': {
+                'process_id': 'aggregate_spatial','arguments': {
+                    'data': {'from_node': 'renamelabels1'},'geometries': {
+                        'type': 'GeometryCollection','geometries': [{
+                                                                        'type': 'Polygon','coordinates': (((16.138916,
+                                                                                                            48.1386),(
+                                                                                                           16.138916,
+                                                                                                           48.320647),(
+                                                                                                           16.2,
+                                                                                                           48.320647),(
+                                                                                                           16.2,
+                                                                                                           48.1386),(
+                                                                                                           16.138916,
+                                                                                                           48.1386)),)
+                                                                        },{
+                                                                        'type': 'Polygon','coordinates': (
+                            ((16.3,48.1386),(16.3,48.320647),(16.5,48.320647),(16.5,48.1386),(16.3,48.1386)),)
+                                                                        }]
+                        },'reducer': {
+                        'process_graph': {
+                            'mean2': {
+                                'process_id': 'mean','arguments': {'data': {'from_parameter': 'data'}},'result': True
+                                }
+                            }
+                        }
+                    }
+                },'fitclassrandomforest1': {
+                'process_id': 'fit_class_random_forest','arguments': {
+                    'num_trees': 3,'predictors': {'from_node': 'aggregatespatial1'},'target': {
+                        'id': 'labels','data': {
```

Comment:
the `target` parameter should (in this phase) be a FeatureCollection, so `'type': 'FeatureCollection',` should be at top level, there should be no level `'id': 'labels','data': {`